shiny_proxy_jlab_query_parms/LICENSE

```diff
@@ -0,0 +1,29 @@
+BSD 3-Clause License
```

Comment:
One thing, we may have to review the license.

Reply:
Sure. The BSD License comes from the `cookiecutter` boilerplate generator for the extension. As far as I can tell, Apache can subsume BSD3 contributions, so we can remove the sublicense from this extension.